tests/taint_maturity/elixir/taint_seq.ex

```diff
@@ -0,0 +1,34 @@
+defmodule Test do
+
+  def test1() do
+    #ruleid: taint-maturity
+    sink("tainted")
+  end
+
+  def test2() do
+    x = "safe"
+    a = x
+    x = "tainted"
+    y = x
+    z = y
+    #ruleid: taint-maturity
+    sink(z)
```

Comment:
This is the one that is failing because `=` is not being translated as a `VarDef` in the above cases. (In general `=` is like `let-in` in OCaml, so when the LHS is a pattern we could use a `LetPattern` node, as we do in OCaml.)